types/trouter/trouter-tests.ts

```diff
@@ -0,0 +1,52 @@
+import Trouter from 'trouter';
+
+// Default type is "any" for handlers
+const router = new Trouter();
+router.add('get', '/user/:name/:id', 'test', 'test2', 'test3');
```

Comment:
Methods should be capitalized

Reply:
That's actually weird https://github.com/lukeed/trouter .
The docs say `lowercase` but all the code and samples imply uppercased 🤔

Reply:
Whooooops. That's a mistake, thanks!